src/env_cli_vars.nim

```diff
@@ -240,12 +240,12 @@ type StatusDesktopConfig = object
     name: "USE_MOCKED_KEYCARD"
     abbr: "use-mocked-keycard" .}: bool
   httpApiEnabled* {.
-    defaultValue: true
+    defaultValue: false
```

Comment:
Shouldn't these get the default value from `DEFAULT_FLAG_CONNECTOR_ENABLED` because now merely toggling the feature flag will no longer enable the feature itself

Reply:
I'll update that in the master version :+1: